src/SprykerEco/Service/ComputopApi/Mapper/ComputopApiMapper.php

```diff
@@ -134,8 +134,8 @@ public function getDescriptionValue(array $items): string
     public function generateReqIdFromQuoteTransfer(QuoteTransfer $quoteTransfer): string
     {
         return $this->generateReqId(
-            $quoteTransfer->getTotals()->getHash(),
-            $quoteTransfer->getCustomer()->getCustomerReference() ?? static::GUEST_CUSTOMER_REFERENCE
+            $quoteTransfer->getTotalsOrFail()->getHashOrFail(),
```

Comment:
Please make sure that this reflects in the API specification (if needed).  The same for other `get*OrFail`.

Reply:
Fixed